content/index.mdx

```diff
@@ -43,6 +43,12 @@ Relational databases are great at handling highly structured data and making it
 <DocLink icon='file' text="Using the expand and contract pattern for schema changes" href={'/types/relational/expand-and-contract-pattern'}/>
 <DocLink icon='file' text="Comparing common database infrastructure patterns" href={'/types/relational/infrastructure-architecture'}/>
 
+### [Document databases](/types/document)
+
+Document databases are a great way for working flexibily with data with high scalability and agility. Discover why document databases continue gaining in popularity.
```

Comment:
For whatever reason, "continue gaining in" sounds a bit strange to my ear in a way that "continue growing in" or "continue to gain in" does not.  Feel free to ignore but I thought I'd mention it in case it reads that way to you too.